docs/querying/basics.md

```diff
@@ -55,10 +55,19 @@ Example:
 
 ### Float literals
 
-Scalar float values can be literally written as numbers of the form
-`[-](digits)[.(digits)]`.
+Scalar float values can be literally written as integer or floating-point numbers, for example:
 
+    23
     -2.43
+    3.4e-9
+    0x8f
```

Comment:
Is this something we actually support? I think we should stick to basic floats and ints.

Reply:
We do not support that and #5984 made sure we do not scrape that either.

Reply:
Oh wait,yes we support that

Reply:
Indeed, this is not something we support. https://github.com/prometheus/prometheus/blob/master/promql/parser/lex.go#L717

Reply:
In practice: https://prometheus.demo.do.prometheus.io/graph?g0.range_input=1h&g0.expr=0x1&g0.tab=1

Reply:
So it is